CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased] - XXXX-XX-XX
-
+### Changed
+- PySide2 installed by default if wrapped Qt (PyQt5,PySide2) not found (by [#187](https://github.com/cbrnr/mnelab/pull/187))
```

Comment:
```suggestion
- PySide2 installed by default if wrapped Qt bindings (PyQt5, PySide2) are not found ([#187](https://github.com/cbrnr/mnelab/pull/187) by [Guillaume Dollé](https://github.com/gdolle))
```

Reply:
(I let you decide, personally I'm not found of naming authors/url in files except for the AUTHORS.<ext>/CONTRIBUTORS.<ext> files. But is your code ;) )

Reply:
Fair point, I should have asked you if including your name was OK. However, from your comment I assume that it's fine, and I think it's quite useful to find the names with the PRs to acknowledge the authors.

Reply:
Yes no worries, it just a matter of preference.
Talking about changelog, authors, licence file, I just saw that the code is BSD-3 clause, I think it's incompatible with the QT open source  GLP/L-GPL licensing no ? (I'm far from being a license expert though)

Reply:
I'm no expert either, but I think we're allowed to use BSD-3 because (1) it is GPL-compatible (see e.g. [here](https://www.riverbankcomputing.com/pipermail/pyqt/2014-February/033848.html)), (2) we're not including PyQt5/PySide2/Qt in our project, we're just importing it, and (3) PySide2 (and Qt) is available under the LGPL. But let me know if you think I'm wrong.

Reply:
Ok it seems I was wrong BSD Clause 3 is GPL compatible according to [fsf](https://directory.fsf.org/wiki/License:BSD-3-Clause)